docs/gitlab.md

```diff
@@ -0,0 +1,69 @@
+# Gitlab
+## Add gitlab ssh key
+
+We use **ssh** protocol for project clone on gitlab.   
+This is documentation for add ssh key on gitlab account. You can found gitlab doc [here](https://docs.gitlab.com/ee/user/ssh.html).
+
+### Generate an SSH key pair
+
+```bash
+ssh-keygen -t ed25519 -C "<email>"
+```
+
+Press Enter. Output similar to the following is displayed:
+
+```bash
+Generating public/private ed25519 key pair.
+Enter file in which to save the key (/home/user/.ssh/id_ed25519):
+```
+
+Accept the suggested filename and directory unless want to save in a specific directory.
+
+Specify a passphrase:
+
+_Passphrase is not asked on git clone, so you can fill it_
+
+```bash
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+```
+
+A public and private key are generated. Next, add the public SSH key to your GitLab account.
+
```

Comment:
Add the following about how to verify your passphrase :

If already have an ssh key and you wish to test your passphrase you can run the following :

```shell
ssh-keygen -y -f ~/.ssh/id_ed25519
```